### PR TITLE
Added upgrader for ContourLoadSet and UniformLoadSet

### PR DESCRIPTION
**Pull request by JosefTaylor**
### NOTE: Depends on 

[BHoM PR 1229](https://github.com/BHoM/BHoM/pull/1229)
   
### Issues addressed by this PR

Closes [BHoM 1224](https://github.com/BHoM/BHoM/issues/1224)

### CODE REVIEW

File: BHoMUpgrader42/Converter.cs [hunk 1]
```diff
@@ -47,7 +49,73 @@ public Converter() : base()
...
+            {
+                Dictionary<string, object> oldLoadDict = outObject as Dictionary<string, object>;
+
+                if (oldVersion.TryGetValue("Loadcases", out outObject))
+                {
+                    Dictionary<string, object> oldLoadcaseDict = outObject as Dictionary<string, object>;
+                    List<string> names = oldLoadDict.Keys.ToList();
+
+                    newVersion["Loads"] = names.Select(x => new Dictionary<string, object>()
+                    {
```
Comment (IsakNaslundBh) [inline BHoMUpgrader42/Converter.cs:78]: think you might need to add a `"_t", _fullPathToType_` here as well

File: BHoMUpgrader42/Converter.cs [hunk 2]
```diff
@@ -47,7 +48,57 @@ public Converter() : base()
...
+                        { "_t", "BH.oM.Structure.Loads.UniformLoadSetRecord" },                   
+                        { "Name", x },
+                        { "Loadcase", oldLoadcaseDict[x] },
+                        { "Load", oldLoadDict[x] },
+                }).ToList();
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Converter.cs:84]: This line needs indenting one extra indent to line up correctly with the bits above it.

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by JosefTaylor** [comment]
@BHoMBot check installer

**Comment by JosefTaylor** [comment]
@BHoMBot check code-copyright

**Comment by JosefTaylor** [comment]
@BHoMBot check copyright

**Comment by JosefTaylor** [comment]
@BHoMBot check copyright-compliance

**Comment by JosefTaylor** [comment]
@FraserGreenroyd would you mind dealing with the check-installer and code-copyright tests? They appear to be superseded by other tests, and I don't know how to trigger them or dismiss them.

**Comment by maryannewachter** [comment]
@JosefTaylor Versioning does not appear to have worked for the contour load set (though it did for the uniform load set). I'm getting the generic "This is a magic component" message

image

**Comment by JosefTaylor** [comment]
@FraserGreenroyd @IsakNaslundBh Could you please add your review and merge if satisfied? @maryannewachter has approved the RAM PR including the functionality of the migrated objects.

**Review by maryannewachter — APPROVED** [review]
Works per [review](https://github.com/BHoM/RAM_Toolkit/pull/183) of RAM_oM

**Comment by IsakNaslundBh** [comment]
@BHoMBot check core

**Pull request merged**